### PR TITLE
 block_copy: Use QemuImg instead of generating qemu-img cmd

### PR DESCRIPTION
**Pull request by vivianQizhu**
id: 1517110
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
15:10:23 INFO | PASS 1-smp_4.1024m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.x86_64.myprovider.block_stream.check_backingfile

File: qemu/tests/block_copy.py [hunk 1]
```diff
@@ -282,9 +283,9 @@ def get_backingfile(self, method="monitor"):
         if method == "monitor":
             return self.vm.monitor.get_backingfile(self.device)
 
-        qemu_img = utils_misc.get_qemu_img_binary(self.params)
-        cmd = "%s info %s " % (qemu_img, self.get_image_file())
-        info = utils.system_output(cmd)
+        qemu_img = qemu_storage.QemuImg(self.params, self.data_dir, self.tag)
+        qemu_img.image_filename = self.get_image_file()
+        info = qemu_img.info()
```
Comment (yanan-fu) [inline qemu/tests/block_copy.py]: `force_share = True`  ?

Comment (vivianQizhu) [reply qemu/tests/block_copy.py]: @yanan-fu Ah...yes, I have tested on qemu2.9, therefore passed, thanks for pointing out.

**Comment by yanan-fu** [comment]
@vivianQizhu  For new qemu version,  without `force_share = True`, the test result should be be pass.
Please be sure,  qemu version is high than `qemu-kvm-rhev-2.10.0-3.el7`  and kernel version is higher than ``
So, the result is valid.

**Comment by yanan-fu** [comment]
@vivianQizhu For new qemu version, without force_share = True, the test result should be be pass.
Please be sure, qemu version is high than qemu-kvm-rhev-2.10.0-3.el7 and kernel version is higher than kernel-3.10.0-704.el7
So, the result is valid.

**Comment by yanan-fu** [comment]
LGTM
Acked-by: Yanan Fu <yfu@redhat.com>

**Comment by vivianQizhu** [comment]
Retest with force_share=True on qemu-kvm-rhev-2.10.0-6, passed.
16:13:19 INFO | PASS 1-smp_4.2048m.repeat1.run_test.Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.myprovider.block_stream.check_backingfile

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**